packages/builder/src/components/common/HelpMenu.svelte

```diff
@@ -1,11 +1,12 @@
 <script>
   import FontAwesomeIcon from "./FontAwesomeIcon.svelte"
   import { Popover, Heading, Body } from "@budibase/bbui"
-  import { licensing } from "stores/portal"
   import { isEnabled, TENANT_FEATURE_FLAGS } from "helpers/featureFlags"
+  import { auth } from "stores/portal"
```

Comment:
What's the difference between using the `licensing` store and the `auth` store?

Reply:
Not really sure on this, you can output the same thing using either. Same character count which I found freaky. 

The reason I used auth over licensing is due to seeing it being done and used elsewhere in the codebase. `usage.svelte` uses it for the plan type.

```
$licensing?.license.plan.type
$auth.user?.license.plan.type
```

I've asked one of the engineers and they have said to use licensing over auth as its newer and going forward thats the store that will be used to get the plan types.

I'll update it to use this.